primefaces/src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js

```diff
@@ -121,8 +121,7 @@ PrimeFaces.widget.AutoComplete = PrimeFaces.widget.BaseWidget.extend({
         this.cfg.pojo = this.hinput.length == 1;
         this.cfg.minLength = this.cfg.minLength != undefined ? this.cfg.minLength : 1;
         this.cfg.cache = this.cfg.cache || false;
-        this.cfg.resultsMessage = this.cfg.resultsMessage || ' results are available, use up and down arrow keys to navigate';
-        this.cfg.ariaEmptyMessage = this.cfg.emptyMessage || 'No search results are available.';
+        this.cfg.ariaEmptyMessage = this.cfg.emptyMessage;
```

Comment:
`this.cfg.resultsMessage` is already used to display the message? (I see we do the fallback in Java now).